gulp/tasks/eslint.js

```diff
@@ -8,23 +9,18 @@ const argv = require('yargs').argv;
 const gulpif = require('gulp-if');
 const eslint = require('gulp-eslint');
 const config = require('../config');
+const eslintConfig = require('eslint-config-actum').getConfig({ environment });
 
 /* Plugins */
```

Comment:
It will be removed in new configuration. These comments are still left on the master and I created this branch from master. Do you suggest removing them here too?